assets/src/frontend/components/Home.vue

```diff
@@ -41,6 +41,9 @@
                         <span class="toggle-icon grid-view flaticon-menu" @click="productView = 'grid'" :class="{ active: productView == 'grid'}"></span>
                     </div>
                 </div>
+                <div class="progress" v-show="!allProductsLoaded && page > 1">
+                    <progress v-bind:max="totalPages + 1" v-bind:value="page" />
```

Comment:
For consistency, we can use the shorthand `v-bind` here.

Reply:
Good one. Fixed!